GoodMorning/GoodMorning/Extensions/UIStackVeiw+.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  UIStackVeiw+.swift
+//  GoodMorning
+//
+//  Created by Sunny on 2023/08/10.
+//
+
+import UIKit
+
+extension UIStackView {
+    func addArrangedSubviews(_ views: [UIView]) {
+        views.forEach { self.addArrangedSubview($0) }
+    }
+}
```

Comment:
```suggestion
extension UIStackView {

    func addArrangedSubviews(_ views: [UIView]) {
        views.forEach { self.addArrangedSubview($0) }
    }
    
}
```
요렇게 개행처리하면 좋을 거 같아요!!😄

Reply:
앗,, ㅎㅎ 수정 완료했습니다!

> 커밋: 359a8d53b70bc313c0db2df71c1294a0b23b6b17